lib/modules/datasource/metadata-manual.ts

```diff
@@ -54,6 +54,10 @@ export const manualChangelogUrls: Record<string, Record<string, string>> = {
     neo4j: 'https://neo4j.com/release-notes/',
     'whitesource/renovate': 'https://github.com/whitesource/renovate-on-prem',
   },
+  gradle: {
+    'com.figure.gradle.semver-plugin': 
```

Comment:
There is no `gradle` datasource in Renovate. I think you'll find this is `maven`. Is this the full package name, without a scope?

Reply:
@viceice made some suggested changes that I've committed to fix the `maven`.  

Technically the package name is `com.figure.gradle:semver-plugin`, but the plugin identifier is `com.figure.gradle.semver-plugin`. Which one does renovate expect?

Since it's looking at maven stuff, I assume it'll be looking at the normal java pom file (and not the gradle pom file):

```
<project ...>
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.figure.gradle</groupId>
  <artifactId>semver-plugin</artifactId>
  <version>1.1.0</version>
  <dependencies>
  </dependencies>
</project>
```

Should I change this to the true maven package name?

Reply:
Have a look in your debug logs for `packageFiles with updates`, then find this dependency. Look at the `packageName` field.